worlds/dark_souls_3/data/items_data.py

```diff
@@ -154,8 +198,8 @@
     "Soul of a Demon": 0x400002E3,
 
     # Upgrade materials
-    **{"Titanite Shard #"+str(i): 0x400003E8 for i in range(1, 11)},
-    **{"Large Titanite Shard #"+str(i): 0x400003E9 for i in range(1, 11)},
+    **{"Titanite Shard #"+str(i): 0x400003E8 for i in range(1, 16)},
+    **{"Large Titanite Shard #"+str(i): 0x400003E9 for i in range(1, 16)},
```

Comment:
You can't add 5 Titanite Shards and 5 Large Titanite Shards in the middle of the goods_table or it will shift every IDs.

You should write the following two lines at the end of the table
"**{"Titanite Shard #"+str(i): 0x400003E8 for i in range(11, 16)},"
"**{"Large Titanite Shard #"+str(i): 0x400003E9 for i in range(11, 16)},"